samples/subsys/bindesc/bindesc.rst

```diff
@@ -0,0 +1,10 @@
+.. _bindesc_samples:
```

Comment:
needs a sample.yaml file - not blocking this PR as it is now but please ensure this compiles on a board you have locally, then submit a PR adding a sample.yaml file after this is merged